### ISSUE TITLE
How to load the pretrained weights file for FaceNet?

### ISSUE THREAD

**Issue by miranthajayatilake** [JudasDie/deeplearning.ai#4]
I'm trying out Facenet code locally, and having difficulties loading the pretrained model.
It's giving an error in the load_weights function in the fr_utils.py as below;
`conv_w = genfromtxt(paths[name + '_w'], delimiter=',', dtype=None)
KeyError: 'conv1_w'`

Can you help to sort this out?
Thanks.

**Comment by viethungtsdv** [comment]
I have same problem with you? Please share how to fix it. Thanks

**Comment by Panderio** [comment]
I have the same problem can someone help

**Comment by zeintiz** [comment]
I have the same problem, so could you please tell me how you fix this problem . thankx in advance.

> I'm trying out Facenet code locally, and having difficulties loading the pretrained model.
> It's giving an error in the load_weights function in the fr_utils.py as below;
> `conv_w = genfromtxt(paths[name + '_w'], delimiter=',', dtype=None) KeyError: 'conv1_w'`
> 
> Can you help to sort this out?
> Thanks.

**Comment by InfinityQuest** [comment]
Hello, I faced the same problem...What I did was I opened the weight folder and realized that some weights ending with conv2_w and conv_w are still in zip format or unreadable. So, when I ran the code and get above error, I note down the file which is causing this error. Then from this link http://bggit.ihub.org.cn/p30597648/facenet-face-recognition/tree/master/weights download the said weight raw file and copy it to the wieght folders.. Repeated the procedure until all the files causing this error is copied. Thats it. The face net program moves ahead from this problem.

Regards

I am a newbie ...so bear with my procedure
